### PR TITLE
MM-27450 Remove remaining references to jQuery 🎉

### PR DESCRIPTION
**Pull request by hmhealey**
I was sad to see that Jesús had removed all of jQuery without me having a chance to get involved, so I'm glad to have found a way to contribute to that

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-27450

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/9409

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
YES

**Comment by jespino** [comment]
I tried that, but I think bootstrap was a problem because depends on jQuery being present.

**Comment by hmhealey** [comment]
It turns out we can just fix that by not importing Bootstrap 😀

We still rely on Bootstrap's CSS in a some places, but it looks like we haven't relied on the JS part of it since we finally got everything switched over to React-Bootstrap at some point. Comparing this test server with one on my other PR, we're reducing the bundle size by about 40KB here too

**Review by jespino — APPROVED** [review]
Love it!!

File: package.json [hunk 1]
```diff
@@ -39,7 +39,6 @@
     "html-to-react": "1.4.5",
     "inobounce": "0.2.0",
     "jasny-bootstrap": "3.1.3",
-    "jquery": "3.6.0",
```
Comment (M-ZubairAhmed) [inline package.json:42]: This single line of code gives me so much happiness 😊

**Pull request merged**